### PR TITLE
ISPN-7669 Evaluate DataContainer size method invocations to make sure

### PR DESCRIPTION
**Pull request by wburns**
they make sense

* Changed size call to use sizeIncludingExpired
* Refactor size method to use another option when possible

https://issues.jboss.org/browse/ISPN-7669

### CODE REVIEW

File: core/src/main/java/org/infinispan/eviction/impl/PassivationManagerImpl.java [hunk 1]
```diff
@@ -99,13 +99,16 @@ public void passivateAll() throws PersistenceException {
       if (enabled && !skipOnStop) {
          long start = timeService.time();
          log.passivatingAllEntries();
+         int count = 0;
          for (InternalCacheEntry e : container) {
+            if (count != Integer.MAX_VALUE) {
+               count++;
```
Comment (danberindei) [inline core/src/main/java/org/infinispan/eviction/impl/PassivationManagerImpl.java]: Could use a `long` instead.

Comment (wburns) [inline core/src/main/java/org/infinispan/eviction/impl/PassivationManagerImpl.java]: That is what I had originally, but the passivatedEntries log method requires int. Want me to update both?

**Comment by wburns** [comment]
Updated.

**Comment by wburns** [comment]
If we need to @tristantarrant this can be delayed to 9.1, but it is pretty trivial and should help performance a bit for all bulk operations.

**Comment by tristantarrant** [comment]
Merged to master, thanks

**Pull request merged**